src/OllamaApiClient.cs

```diff
@@ -249,50 +254,50 @@ private async Task<TResponse> PostAsync<TRequest, TResponse>(string endpoint, TR
 
 		await EnsureSuccessStatusCode(response);
 
-		await foreach (var result in ProcessStreamedResponseAsync<TResponse>(response, cancellationToken))
+		await foreach (var result in ProcessStreamedResponseAsync<TResponse>(response, cancellationToken, IncomingJsonSerializerOptions))
 			yield return result;
 	}
 
-	private static async IAsyncEnumerable<TLine?> ProcessStreamedResponseAsync<TLine>(HttpResponseMessage response, [EnumeratorCancellation] CancellationToken cancellationToken)
+	private static async IAsyncEnumerable<TLine?> ProcessStreamedResponseAsync<TLine>(HttpResponseMessage response, [EnumeratorCancellation] CancellationToken cancellationToken, JsonSerializerOptions? options = default)
```

Comment:
Since these are both deserializing to json, it's always the "incoming" seriaizer options here. As far as I can see, there's no need to pass these in as arguments.

Reply:
OK I will remove the static modifier then